site/content/docs/cli/constructs-reference.md

```diff
@@ -431,33 +430,83 @@ new ApiCheck('check-group-api-check-1', {
-- `locations`: An array of location codes where to run the Checks in the group, i.e. `['us-east-1', 'eu-west-1']`.
-- `privateLocations`: An array of [Private Locations](/docs/private-locations/) slugs, i.e. `['datacenter-east-1']`.
-- `alertChannels`: An array of `AlertChannel` objects to which to send alert notifications.
-- `activated`: A boolean value if all the Checks in the group are activated.
-- `muted`: A boolean value if alert notifications from the Checks in the group are muted, i.e. not sent out.
-- `tags`: An array of tags. Group tags trickle down to tags on the individual Checks. i.e. `['product', 'api']`
-- `runtimeId`: The ID of which [runtime](/docs/runtimes/specs/) to use for the Checks in the group.
-- `environmentVariables`: An array of objects defining variables in the group scope, i.e. `[{ key: 'DEBUG', value: 'true', secret: true | locked: true }]`
+- `locations`: An array of one or more [public locations](/docs/monitoring/global-locations) where Checks in this group should run (e.g. `['us-east-1', 'eu-west-1']`). If this or `privateLocations` is set, it overrides the location settings of all Checks in the group.
+- `privateLocations`: An array of one or more [private locations](/docs/private-locations) where Checks in this group should run (e.g. `['datacenter-east-1']`). If this or `locations` is set, it overrides the location settings of all Checks in the group.
+- `alertChannels`: An array of [AlertChannel](#alertchannel) objects to be alerted on when checks in this group fail or recover.
+- `tags`: An array of tags i.e. `['product', 'api']`. Group tags trickle down to tags on the individual Checks.
+- `runtimeId`: The ID of which [runtime](/docs/runtimes/specs/) to use for the Checks in the group. Use this if the checks in this group require a different runtime than your account default.
+- `environmentVariables`: An array of objects defining [environment variables in the group scope](/docs/groups/variables), i.e. `[{ key: 'DEBUG', value: 'true', secret: true | locked: true }]`.
 - `localSetupScript`: Any JS/TS code as a string to run before each API Check in this group.
 - `localTearDownScript`: Any JS/TS code as a string to run after each API Check in this group.
-- `retryStrategy`: A [RetryStrategy](#retrystrategy) object configuring [retries](/docs/alerting-and-retries/) for failed check runs.
-- `apiCheckDefaults`: A set of defaults for API Checks. This should not be needed. Just compose shared defaults using JS/TS.
+- `retryStrategy`: A [RetryStrategy](#retrystrategy) object configuring [retries](/docs/alerting-and-retries/) for failed Check runs. If set, all checks in the group use the group's retry strategy. If not set, individual Check settings are used.
+- `apiCheckDefaults`: A set of [defaults for API Checks](/docs/groups/api-check-defaults/). This should not be needed. Just compose shared defaults using JS/TS.
```

Comment:
what's apiCheckDefaults? given this text I'd deprecate this option entirely. Such a funny paragraph you run into :) 

```suggestion
- `apiCheckDefaults`: Deprecated. [A set of shared values for API Checks](/docs/groups/api-check-defaults/).  Instead, use JavaScript or TypeScript to define shared defaults directly in your code.
```
we'd also need to deprecate in the docs page 😅

Reply:
Yeah so I was waiting for Simo's input on this but maybe you can help me out here as well - How come they are deprecated? We continue to support them in the UI...

![Uploading Screenshot 2025-06-06 at 14.59.06.png…]()

Reply:
they're not deprecated as far as I can see - I'm proposing that they do become deprecated based on the docs saying "you don't need this, just use JS/TS code to do it" - I'd check usage before doing such deprecation, but still, felt inconsistent